src/nova-driver.md

```diff
@@ -0,0 +1,54 @@
+# Nova GPU Driver
+
+Nova is a driver for GSP-based Nvidia GPUs that is currently under development
+and is being written in Rust.
+
+As of today, the driver is in the (very) early stage and not yet operational.
+
+
+## Contact
+
+To contact the team and / or participate in development, please use the
+mailing list of the nouveau-driver for now: nouveau@lists.freedesktop.org
+
+
+## Resources
+
+- [Official Source Tree](https://gitlab.freedesktop.org/drm/nova)
+- [Announcement E-Mail](https://lore.kernel.org/dri-devel/Zfsj0_tb-0-tNrJy@cassiopeiae/)
+
+In the source tree, the driver lives in `drivers/gpu/drm/nova`
+
+
+## Utilized Common Rust Infrastructure
+
+Currently, Nova is based on those downstream branches:
+
+- rust-common
+- rust-device
+- rust-drm
+- rust-pci
+
```

Comment:
I think we should just link to R4L's topic branch section and add the corresponding branches there. If we do that, I'm not sure we even need a separate "Utilized Common Rust Infrastructure" section anymore.

Reply:
> and add the corresponding branches there

Do you mean creating a separate list there?

I'd keep the list here and link the R4L branch section.

The R4L branch section is "generic" and has nothing to do with Nova, so why should there be a list describing our payload

Reply:
Perhaps Danilo meant something like adding Nova to the list of users to each of the branches there? i.e. like we have for `rust-pci` where we mention the NVMe driver.

> I'd keep the list here and link the R4L branch section.

Sounds good too, especially if you want to give more details for each of the branches here on how Nova uses them (e.g. like Danilo mentioned in the announcement).

Reply:
I think it's better to have the info about the branches here, so that people pulling or cherry picking around can see here immediately what they need to build Nova.
ACK, @dakr ?

Reply:
IMHO it's enough to say that Nova is based on R4L staging/* branches and just add a link to the R4L topic branch section. If you want to be explicit about which of those exactly we depend on, you can surely add this here as well.